tests/integration/devfile/cmd_devfile_url_test.go

```diff
@@ -111,9 +112,11 @@ var _ = Describe("odo devfile url command tests", func() {
 
 			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
 			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(commonVar.Context, "devfile.yaml"))
-
-			helper.CmdShouldPass("odo", "url", "create", url1, "--host", host, "--ingress")
+			// we check the default url
 			stdout := helper.CmdShouldPass("odo", "url", "list")
```

Comment:
This will fail on a kubernetes cluster
```
[odo]  ✗  no URLs found for component rgsrbu. Refer `odo url create -h` to add one
Deleting project: gqpnssmcal
Running kubectl with args [kubectl delete namespaces gqpnssmcal]
[kubectl] namespace "gqpnssmcal" deleted
Setting current dir to: /home/travis/gopath/src/github.com/openshift/odo/tests/integration/devfile
```
https://travis-ci.com/github/openshift/odo/jobs/399332372#L1459

Reply:
Resolved